src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -152,9 +149,25 @@ protected virtual CodeActionPriority ComputePriority()
         /// </summary>
         public virtual ImmutableArray<string> Tags => ImmutableArray<string>.Empty;
 
-        internal virtual ImmutableArray<CodeAction> NestedCodeActions
+        /// <summary>
+        /// Child actions contained within this <see cref="CodeAction"/>.  Can be presented in a host to provide more
+        /// potential solution actions to a particular problem.  To create a <see cref="CodeAction"/> with nested
+        /// actions, use <see cref="Create(string, ImmutableArray{CodeAction}, bool)"/>.
+        /// </summary>
+        public virtual ImmutableArray<CodeAction> NestedActions
             => ImmutableArray<CodeAction>.Empty;
 
+        /// <summary>
+        /// If this code action contains <see cref="NestedActions"/>, this property provides a hint to hosts as to
+        /// whether or not it's ok to elide this code action and just present the nested actions instead.  When a host
+        /// already has a lot of top-level actions to show, it should consider <em>not</em> inlining this action, to
+        /// keep the number of options presented to the user low.  However, if there are few options to show to the
+        /// user, inlining this action could be beneficial as it would allow the user to see and choose one of the
+        /// nested options with less steps.  To create a <see cref="CodeAction"/> with nested actions, use <see
+        /// cref="Create(string, ImmutableArray{CodeAction}, bool)"/>.
+        /// </summary>
+        public virtual bool IsInlinable => false;
```

Comment:
this is part and parcel of NestedActions (and is also part of the Create method).